python/ray/_private/runtime_env/packaging.py

```diff
@@ -579,7 +582,9 @@ def download_and_unpack_package(
             protocol, pkg_name = parse_uri(pkg_uri)
             if protocol == Protocol.GCS:
                 # Download package from the GCS.
-                code = _internal_kv_get(pkg_uri)
+                code = await gcs_aio_client.internal_kv_get(
+                    pkg_uri.encode(), namespace=None, timeout=None
```

Comment:
should we give a namespace to the PKGs?

Reply:
We didn't have one before, but I guess you're right, it's safer to use a namespace like we do in Serve.  Let me update the PR